internal/cmd/auth_test.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.flipt.io/flipt/internal/config"
+	"go.uber.org/zap/zaptest"
+)
+
```

Comment:
I started adding the jwks test here but realized it would be easier in the `middleware_test.go` that I added the other new tests in.. but figured it didn't hurt to start adding tests for `authenticationGRPC` anyways